Miscellaneous/VerificationAndValidationPlan.tex

```diff
@@ -0,0 +1,57 @@
+\documentclass[11pt,fleqn]{article}
+
+\usepackage{blindtext}
+\usepackage{hyperref}
+\hypersetup{
+    colorlinks=true,
+    linkcolor=blue,
+    filecolor=magenta,      
+    urlcolor=cyan,
+    pdftitle={Overleaf Example},
+    pdfpagemode=FullScreen,
+    }
+\urlstyle{same}
+
+\newcommand{\indentpar}{\phantom{=}}
+\newcommand{\bli}{\begin{itemize}}
+\newcommand{\eli}{\end{itemize}}
+
+\begin{document}
+
+    %% Title Page %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+      
+      \title{Verification and Validation Plan for VDisp Software}
+      \date{May 16th, 2022}
+      \author{Emil Soleymani, Dr.~Spencer Smith\\ McMaster University}
+      \maketitle
+
+      \medskip
+
+      % Description
+      \indentpar In order to verify the validity of the \emph{VDisp} software, we came
```

Comment:
"In  order to" should be replaced by "to".  I mention this so that you can get out of the habit of using it.  :smile:  Other low information content phrases can be found in the link given in my [writing checklist](https://gitlab.cas.mcmaster.ca/smiths/cas741/-/blob/master/BlankProjectTemplate/docs/Writing-Checklist.pdf).